addons/im_livechat/static/src/embed/frontend/livechat_root.js

```diff
@@ -18,7 +18,7 @@ export class LivechatRoot extends Component {
     static props = {};
 
     setup() {
-        useSubEnv({ inShadow: true });
+        useSubEnv({ rootId: "liveChatRoot" });
```

Comment:
I don't like this magic string. We could just pass `shadwoRootId` as a prop  when mounting the app in the `boot_service`